src/org/labkey/test/components/ui/grids/EditableGrid.java

```diff
@@ -1096,6 +1096,9 @@ public List<String> getColumnNames()
 
         public WebElement inputCell()
         {
+            // Wait for the input to appear because there are some cases where we type into a cell to trigger the input
+            // to appear, and then immediately try to type into the input and it may not be there.
+            Locators.inputCell.waitForElement(table, WAIT_FOR_JAVASCRIPT);
```

Comment:
you can return the value of `Locators.inputCell.waitForElement(table, WAIT_FOR_JAVASCRIPT)` here
waiting for it is more robust than using .findElement in the sense that it will tolerate it if the element isn't already there by the time the test thread arrives.